docs/general/testing/web/index.md

```diff
@@ -26,3 +26,57 @@ Each PR of [Jellyfin Web](https://github.com/jellyfin/jellyfin-web) and [Jellyfi
+
+Jellyfin web clients can be hosted as a standalone application without being associated with a Jellyfin server. In that case, a separate Jellyfin server will need to be prepared.
+
+"Test server" below refers to the device hosting the web client in the test setup.
+
+### Prerequisites
+
+Below is a list of things to prepare before testing Jellyfin web clients.
+
+- Have an instance of Jellyfin Server. A dedicated testing setup is recommended.
+- Install [NodeJS, npm and nvm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) on the test server.
+- Install [Git](https://github.com/git-guides/install-git) on the test server.
+- (Optional) Install [GitHub CLI](https://cli.github.com/) or [GitHub Desktop](https://github.com/apps/desktop) on the test server
+- (Optional) A text editor or web IDE to make changes during testing, eg. [VSCode](https://code.visualstudio.com/), [Notepad++](https://notepad-plus-plus.org/) ,[Jetbrains Webstorm](https://www.jetbrains.com/webstorm/).
+
+:::note
+Installing Github CLI or Github Desktop will automatically install Git on the system.
+:::
+
+### Obtaining Source Code
+
+The source code of the web clients can simply be cloned from their respective GitHub repositories: [Jellyfin Web](https://github.com/jellyfin/jellyfin-web/), [Jellyfin Vue](https://github.com/jellyfin/jellyfin-vue/)
+Instructions on how to do so can be found [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository).
+
+### Checking out branches
+
+The `master` branch is where the unstable build is built from. If you wish to test over there, please checkout that branch. To checkout the `master` branch, simply do the following.
+
+- Git CLI, GitHub CLI: open a terminal in the web folder and run `git checkout master`.
+- GitHub Desktop: Select the `master` branch in the branch dropdown menu.
+
+### Checking out PRs
+
+PRs are special branches submitted by contributors with the goal of eventually being merged into `master`. To checkout a PR, please follow the instructions below.
+
+1. Find the PR you would like to checkout in the list of open PRs: [Jellyfin Web](https://github.com/jellyfin/jellyfin-web/pulls), [Jellyfin Vue](https://github.com/jellyfin/jellyfin-vue/pulls).
+2. Checkout the PR in Git: [Git CLI and GitHub CLI](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally), [GitHub Desktop](https://docs.github.com/en/desktop/working-with-your-remote-repository-on-github-or-github-enterprise/viewing-a-pull-request-in-github-desktop)
+
+### Install Dependencies
+
+1. Navigate to the web directory (the folder containing the cloned contents of the repository)
+2. Open a terminal window in the folder.
+3. Run `npm install` to install dependencies. This will need to be done every time you switch to a new branch, as different branches might have different versions of dependencies.
```

Comment:
I'd remove this section and link to the documentation of Web and Vue, since this is perhaps the section more likely to be changed in the future (while the others are less likely to). Things like setting up a monorepo, changing package manager, etc... Will outdate this easily. This, in fact, is not applicable to Vue (you must `cd frontend` first).

I'm also not sure the next one is needed as well for the same reasons. Only a small port change (e.g: Web switches to Vite and the new port is used instead of the old one) and this is out of date.

All this maybe could be solved by using a sentence similar to the last one applicable for Vue, but for web, linking to the repo where info is probably going to be more up to date.

I leave this up to you/other reviewers though. I also understand my concerns are more applicable to Vue in this case (since it usually live sin the bleeding edge) than Web.